nixos/modules/services/web-servers/apache-httpd/default.nix

```diff
@@ -113,26 +113,37 @@ let
     }
     ++ cfg.extraModules;
 
-  loggingConf = (
-    if cfg.logFormat != "none" then
+  loggingFormatConf = (
+    if
```

Comment:
not a blocker: would be less confusing if the `logFormat = "none"` case was handled here along with everything else.